distributed/http/scheduler/prometheus/semaphore.py

```diff
@@ -49,15 +49,14 @@ def collect(self):
         )
 
         semaphore_average_pending_lease_time = GaugeMetricFamily(
-            self.build_name("average_pending_lease_time"),
+            self.build_name("average_pending_lease_time_seconds"),
             "Exponential moving average of the time it took to acquire a lease "
             "per semaphore\n"
             "Note: This only includes time spent on scheduler side, "
             "it does not include time spent on communication.\n"
             "Note: This average is calculated based on order of leases instead "
             "of time of lease acquisition.",
             labels=["name"],
```

Comment:
```suggestion
            labels=["name"],
            unit="seconds",
```
Cosmetic nitpick. The output name remains the same